63070: Custom Domain UI Changes (#812)
replicatedhq/replicated-docs: docs/vendor/packaging-private-images.md

* [Push Images to the Replicated Private Registry](#push-images-to-the-replicated-private-registry)
* [Using Private Registries for helm CLI Installations (Beta)](helm-image-registry)

> If you use the Replicated private registry and proxy service, you can configure custom domains as aliases to bring the Replicated domains inside a customer security review and reduce exposure. This feature is configured at the Team level. For more information about custom domains, see [Using Custom Domains](packaging-private-registry-cname).

Reviewer: ```suggestion
If you use the Replicated private registry or proxy service, you can configure custom domains to alias registry.replicated.com and proxy.replicated.com, to remove the Replicated domains from a customer security review and reduce exposure. This feature is configured at the Team level. For more information about custom domains, see [Using Custom Domains](packaging-private-registry-cname).
```
